views/homepage.handlebars

```diff
@@ -1,9 +1,72 @@
-{{#each Search}}
-<div class="mx-auto justify-center movie sm:h-400 lg:h-530 w-375 text-center">
-    <img class="mx-auto sm:h-1/2 md:h-5/6 lg:h-5/6 w-5/6"
-        src= {{Poster}}>
-    <button class="mx-auto my-auto to-watch">To Watch</button>
-    <button class="mx-auto my-auto watched">Watched</button>
-</div>
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Movie Library</title>
+    <link rel="stylesheet" href="/assets/css/delete-btn.css" />
+    <link rel="stylesheet" href="/assets/css/style.css" />
+</head>
 
-{{/each}}
```

Comment:
Same for this, a little confused why this is being removed  and just want to make sure it's right.

Reply:
Yeah, I had to change homepage.handlebars and search.handlebars.  I'll be adding more to homepage tomorrow.